<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="DataGridViewDataErrorEventArgs.xml" source-language="en-US" target-language="it-IT">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac507424dd07417db23cdbec88dd22f5097f25643bc.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">07424dd07417db23cdbec88dd22f5097f25643bc</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>Provides data for the <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph> event.</source>
          <target state="translated">Fornisce dati per l'evento <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>Handling the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> event lets you handle exceptions thrown by code outside your control (for example, by an external data source).</source>
          <target state="translated">La gestione di <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> evento consente di gestire le eccezioni generate dal codice all'esterno del controllo (ad esempio, un'origine dati esterna).</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>Use the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A&gt;</ph> property to determine the state of the <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> at the time of the exception.</source>
          <target state="translated">Utilizzare il <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A&gt;</ph> proprietà per determinare lo stato del <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> al momento dell'eccezione.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>Use the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> property to retrieve the exception data.</source>
          <target state="translated">Utilizzare il <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> proprietà per recuperare i dati dell'eccezione.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>If you want to handle the exception by additional event handlers, set the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> property to <ph id="ph2">`true`</ph>.</source>
          <target state="translated">Se si desidera gestire l'eccezione dai gestori eventi aggiuntivi, impostare il <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> proprietà <ph id="ph2">`true`</ph>.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>The <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A&gt;</ph> and <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.RowIndex%2A&gt;</ph> properties normally indicate the cell in which the data error occurred.</source>
          <target state="translated">Il <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A&gt;</ph> e <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.RowIndex%2A&gt;</ph> indicano in genere la cella in cui si è verificato l'errore dei dati.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>When the error occurs in an external data source, however, the data source may not provide the column in which the error occurred.</source>
          <target state="translated">Quando si verifica l'errore in un'origine dati esterna, tuttavia, l'origine dati potrebbe non fornire la colonna in cui si è verificato l'errore.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>In this case, the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A&gt;</ph> property typically indicates the column of the current cell at the time of the error.</source>
          <target state="translated">In questo caso, il <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewCellCancelEventArgs.ColumnIndex%2A&gt;</ph> proprietà indica in genere la colonna della cella corrente al momento dell'errore.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>The following code example demonstrates how to respond to information provided by the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs&gt;</ph> class.</source>
          <target state="translated">Esempio di codice riportato di seguito viene illustrato come rispondere alle informazioni fornite dalla <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs&gt;</ph> classe.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Windows.Forms.DataGridViewDataErrorEventArgs">
          <source>This example is part of a larger example available in the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> class overview topic.</source>
          <target state="translated">Questo esempio fa parte di un esempio più esaustivo disponibile nel <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> Cenni preliminari sulla classe.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>The exception that occurred.</source>
          <target state="translated">Eccezione che si è verificata.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>The column index of the cell that raised the <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph>.</source>
          <target state="translated">Indice di colonna della cella che ha generato l'evento <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>The row index of the cell that raised the <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph>.</source>
          <target state="translated">Indice di riga della cella che ha generato l'evento <ph id="ph1">&lt;see cref="E:System.Windows.Forms.DataGridView.DataError" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>A bitwise combination of <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorContexts" /&gt;</ph> values indicating the context in which the error occurred.</source>
          <target state="translated">Combinazione bit per bit dei valori di <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorContexts" /&gt;</ph> che indica il contesto in cui si è verificato l'errore.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.Windows.Forms.DataGridViewDataErrorEventArgs.#ctor(System.Exception,System.Int32,System.Int32,System.Windows.Forms.DataGridViewDataErrorContexts)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorEventArgs" /&gt;</ph> class.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorEventArgs" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>Gets details about the state of the <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridView" /&gt;</ph> when the error occurred.</source>
          <target state="translated">Ottiene i dettagli sullo stato dell'oggetto <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridView" /&gt;</ph> quando si è verificato l'errore.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>A bitwise combination of the <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorContexts" /&gt;</ph> values that specifies the context in which the error occurred.</source>
          <target state="translated">Combinazione bit per bit dei valori di <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorContexts" /&gt;</ph> che specifica il contesto in cui si è verificato l'errore.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>Members of the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts&gt;</ph> enumeration may be combined using the bitwise <ph id="ph2">`OR`</ph> operator to represent the state of a data-bound <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> when a data error occurred.</source>
          <target state="translated">I membri del <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts&gt;</ph> enumerazione può essere combinato tramite bit per bit <ph id="ph2">`OR`</ph> operatore per rappresentare lo stato di un'associazione a dati <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> quando si è verificato un errore di dati.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>For example, if a user enters an invalid cell value (such as entering a name in a cell that requires a date) and then selects a different cell, the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> will try to commit the invalid cell value.</source>
          <target state="translated">Ad esempio, se un utente immette un valore di cella non valido (ad esempio, immettere un nome di una cella che richiede una data) e quindi seleziona una cella diversa, il <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> tenterà di eseguire il commit del valore di cella non valido.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>When the commit fails, the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> will raise a <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> event whose <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A&gt;</ph> property will have a value of <ph id="ph4">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts.Commit&gt;</ph> and <ph id="ph5">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts.CurrentCellChange&gt;</ph>.</source>
          <target state="translated">Quando il commit ha esito negativo, il <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridView&gt;</ph> genererà un <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> evento il cui <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context%2A&gt;</ph> proprietà avrà un valore di <ph id="ph4">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts.Commit&gt;</ph> e <ph id="ph5">&lt;xref:System.Windows.Forms.DataGridViewDataErrorContexts.CurrentCellChange&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>The following code example demonstrates how to investigate the error context.</source>
          <target state="translated">Esempio di codice riportato di seguito viene illustrato come analizzare il contesto di errore.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Context">
          <source>This example is part of a larger example available in the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> class overview topic.</source>
          <target state="translated">Questo esempio fa parte di un esempio più esaustivo disponibile nel <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> Cenni preliminari sulla classe.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>Gets the exception that represents the error.</source>
          <target state="translated">Ottiene l'eccezione che rappresenta l'errore.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>An <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> that represents the error.</source>
          <target state="translated">Oggetto <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> che rappresenta l'errore.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>To handle the exception by more than one <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph>, throw the exception again at the end of a handler by setting the <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> property to <ph id="ph3">`true`</ph>.</source>
          <target state="translated">Per gestire l'eccezione da più <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph>, generare l'eccezione nuovamente alla fine di un gestore impostando il <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> proprietà <ph id="ph3">`true`</ph>.</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>The following code example demonstrates checking the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> property.</source>
          <target state="translated">Esempio di codice seguente viene illustrato come verificare la <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> proprietà.</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception">
          <source>This example is part of a larger example available in the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> class overview topic.</source>
          <target state="translated">Questo esempio fa parte di un esempio più esaustivo disponibile nel <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> Cenni preliminari sulla classe.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>Gets or sets a value indicating whether to throw the exception after the <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorEventHandler" /&gt;</ph> delegate is finished with it.</source>
          <target state="translated">Ottiene o imposta un valore che indica se generare l'eccezione dopo che il delegato <ph id="ph1">&lt;see cref="T:System.Windows.Forms.DataGridViewDataErrorEventHandler" /&gt;</ph> ne ha ultimata l'elaborazione.</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source><ph id="ph1">&lt;see langword="true" /&gt;</ph> if the exception should be thrown; otherwise, <ph id="ph2">&lt;see langword="false" /&gt;</ph>.</source>
          <target state="translated"><ph id="ph1">&lt;see langword="true" /&gt;</ph> se l'eccezione deve essere generata; in caso contrario, <ph id="ph2">&lt;see langword="false" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>The default is <ph id="ph1">&lt;see langword="false" /&gt;</ph>.</source>
          <target state="translated">Il valore predefinito è <ph id="ph1">&lt;see langword="false" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>Set this property to <ph id="ph1">`false`</ph> if the <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph> has dealt with the exception represented by the <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> property, and you do not want to propagate the exception to another <ph id="ph4">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph> that may handle the <ph id="ph5">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> event.</source>
          <target state="translated">Impostare questa proprietà su <ph id="ph1">`false`</ph> se il <ph id="ph2">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph> ha gestito l'eccezione rappresentata dal <ph id="ph3">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception%2A&gt;</ph> proprietà se non si desidera propagare l'eccezione a un altro <ph id="ph4">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventHandler&gt;</ph> che può gestire il <ph id="ph5">&lt;xref:System.Windows.Forms.DataGridView.DataError&gt;</ph> evento.</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>For debugging purposes, it can be useful to propagate the exception and inspect the stack trace for information regarding the context of the error.</source>
          <target state="translated">A scopo di debug, può essere utile propagare l'eccezione e controllare la traccia dello stack per le informazioni relative al contesto dell'errore.</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>The following code example demonstrates using the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> property to indicate that the exception should not be thrown.</source>
          <target state="translated">Esempio di codice seguente viene illustrato l'utilizzo di <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException%2A&gt;</ph> proprietà per indicare che non deve essere generata l'eccezione.</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>This example is part of a larger example available in the <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> class overview topic.</source>
          <target state="translated">Questo esempio fa parte di un esempio più esaustivo disponibile nel <ph id="ph1">&lt;xref:System.Windows.Forms.DataGridViewComboBoxColumn&gt;</ph> Cenni preliminari sulla classe.</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" uid="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.ThrowException">
          <source>When setting this property to <ph id="ph1">&lt;see langword="true" /&gt;</ph>, the <ph id="ph2">&lt;see cref="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception" /&gt;</ph> property value is <ph id="ph3">&lt;see langword="null" /&gt;</ph>.</source>
          <target state="translated">Durante l'impostazione di questa proprietà su <ph id="ph1">&lt;see langword="true" /&gt;</ph>, il valore della proprietà <ph id="ph2">&lt;see cref="P:System.Windows.Forms.DataGridViewDataErrorEventArgs.Exception" /&gt;</ph> è <ph id="ph3">&lt;see langword="null" /&gt;</ph>.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>